src/commonControls/listControl/ListControlAPL.ts

```diff
@@ -0,0 +1,112 @@
+import i18next from 'i18next';
+import { RequestChangedValueByListAct } from '../..';
+
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+interface APLListItem {
+    primaryText: string;
+}
+
+export namespace ListControlAPLPropsBuiltIns {
```

Comment:
I just noticed that enabled is moved into the APL definition.
Perhaps a little cleaner is the following, but its getting a bit hypothetical.  Only change if you feel that it would be very helpful.

```
apl: {
  enabled: true
  generator: ListControlAPLGenerators.TextList
}
```